### PR TITLE
fix stickness block

### PR DESCRIPTION
**Pull request by minniux**
This dynamic block expects a map

Related issue: https://github.com/sweepbright/infrastructure/issues/440

### CODE REVIEW

File: alb_listener/variables.tf [hunk 1]
```diff
@@ -59,7 +59,7 @@ variable "target_deregistration_delay" {
 
 variable "target_stickiness" {
   description = "List(optional, []): An ALB target_group stickiness block"
-  type        = list(string)
+  type        = list(map(string))
```
Comment (simonrondelez) [inline alb_listener/variables.tf:62]: please also update the documentation for this change

**Review by simonrondelez — APPROVED** [review]

File: alb_listener/variables.tf (hunk 1 above)
Comment (minniux) [reply alb_listener/variables.tf:62]: right, done

Comment (iuriaranda) [reply alb_listener/variables.tf:62]: You'll also need to update the variable description field. I'd remove the `List(optional, []): ` prefix from there imo, it's a bit redundant, just leave it in the README documentation

**Review by iuriaranda — APPROVED** [review]
Just a minor improvement on the variable description field.

**Pull request merged**